### PR TITLE
Dockerizing application

### PR DESCRIPTION
**Pull request by mitas1**

### CODE REVIEW

**Comment by mskrip** [comment]
With these changes I am unable to run it localy (using `$ make run') I get this error
image

**Comment by mitas1** [comment]
Fixed

**Comment by mskrip** [comment]
`$ make migrate` gives me this error:
```sh
python manage.py makemigrations
No changes detected
python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 19, in <module>
    execute_from_command_line(sys.argv)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 83, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/migrations/executor.py", line 20, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/migrations/loader.py", line 52, in __init__
    self.build_graph()
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/migrations/loader.py", line 209, in build_graph
    self.applied_migrations = recorder.applied_migrations()
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/migrations/recorder.py", line 65, in applied_migrations
    self.ensure_schema()
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/migrations/recorder.py", line 52, in ensure_schema
    if self.Migration._meta.db_table in self.connection.introspection.table_names(self.connection.cursor()):
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/backends/base/base.py", line 254, in cursor
    return self._cursor()
  File "/home/mskrip/Documents/school/wtam/django-building-log/src/.venv/lib/python3.6/site-packages/django/db/backends/dummy/base.py", line 20, in complain
    raise ImproperlyConfigured("settings.DATABASES is improperly configured. "
django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value. Check settings documentation for more details.
make: *** [Makefile:15: migrate] Error 1
```

**Comment by mitas1** [comment]
Fixed

File: src/webpack.config.js [hunk 1]
```diff
@@ -5,17 +5,23 @@ var BundleTracker = require('webpack-bundle-tracker')
 var assetsDir = path.resolve('buildinglog', 'assets', 'js');
 var staticDir = path.resolve('buildinglog', 'static', 'buildinglog');
 
+var DEBUG = process.env.NODE_ENV !== 'production';
+
 module.exports = {
     entry: {
         application: path.join(assetsDir, 'application.js')
-    },
+    },
```
Comment (mskrip) [inline src/webpack.config.js]: Whitespace at the end of the line

Comment (mitas1) [reply src/webpack.config.js]: Removed

**Comment by mskrip** [comment]
LGTM

**Pull request merged**